src/core/hooks/use-formts/use-formts.ts

```diff
@@ -23,6 +23,7 @@ import {
   _DescriptorApprox_,
 } from "../../types/form-schema-approx";
 import { impl, opaque } from "../../types/type-mapper-util";
+import { FormValidator } from "../../validation";
```

Comment:
there is some inconsistency in file organisation.

If we were to go with current setup I think validation/types should go into /core/types folder, while createFormValidator should go into /core/builders folder.

But this setup isn't very good or scalable, so maybe we should reorganize files anyway?